apps/ejabberd/src/ejabberd_c2s.erl

```diff
@@ -2955,10 +2953,8 @@ add_timestamp({_,_,Micro} = TimeStamp, Server, Packet) ->
     Time = {D,{H,M,S, Micro}},
     case xml:get_subtag(Packet, <<"delay">>) of
         false ->
-            %% TODO: Delete the next element once XEP-0091 is Obsolete
             TimeStampLegacyXML = timestamp_legacy_xml(Server, Time),
```

Comment:
I think that somebody confused the binding/function name and named the modern format as legacy. It's a good opportunity to fix it :)